src/c4/yml/common.hpp

```diff
@@ -288,7 +288,7 @@ struct RYML_EXPORT LineCol
     //! construct from offset, line and column
     LineCol(size_t o, size_t l, size_t c) : offset(o), line(l), col(c) {}
 };
-static_assert(std::is_trivial<LineCol>::value, "LineCol not trivial");
+static_assert(std::is_trivially_copyable<LineCol>::value && std::is_trivially_default_constructible_v<LineCol> && std::is_trivially_destructible_v<LineCol>, "LineCol not trivial");
```

Comment:
Please don't use the _v traits suffix as in `foo_v<>`; it is valid only from C++17 onwards. Use `foo<>::value` instead.